lib/mix/lib/mix/tasks/test.ex

```diff
@@ -210,10 +210,12 @@ defmodule Mix.Tasks.Test do
 
       mix test --only external
 
-  Which is equivalent to:
+  Which is similar to:
 
       mix test --include external --exclude test
 
+  It differs in that the test suite will fail if no tests are executed when the --only option is used.
```

Comment:
Nitpick: backticks in `--only` :)
```suggestion
  It differs in that the test suite will fail if no tests are executed when 
  the `--only` option is used.
```